client/src/components/ImageScrubber/ImageScrubber.js

```diff
@@ -145,27 +143,27 @@ ImageScrubber.propTypes = {
   order: PropTypes.string.isRequired,
   orderBy: PropTypes.string.isRequired,
   numSelected: PropTypes.number.isRequired,
-  byId: PropTypes.object.isRequired,
+  byId: PropTypes.object,
 }
 
 const mapState = state => {
   const keys = Object.keys(state.trees.data)
   return {
     treesArray: keys.map(id => ({
-      ...state.trees.data[id]
+      ...state.imageScrubber.data[id]
```

Comment:
Switched from using `models/trees` redux store to `models/imageScrubber` for the this page. A lot of the code is similar to the ones used in `models/trees` redux store. 

Since two pages are using `trees`data, it could be worth it to look into having a global store object that is passed to the reducers. This way we could grafting the store based on reducers.

what do you think @misterinterrupt @thiagodemellobueno @deepwinter?

ref:
[shaping redux store like db](https://hackernoon.com/shape-your-redux-store-like-your-database-98faa4754fd5)